README.md

```diff
@@ -4,6 +4,14 @@
 
 ![screenshot](https://user-images.githubusercontent.com/77942746/148373306-0321ca6e-4c2f-4429-8e00-2198911fae96.png)
 
+(https://user-images.githubusercontent.com/77942746/148476473-fc69f6c9-066e-41e5-be52-7f538688aedb.png)
+
+(https://user-images.githubusercontent.com/77942746/148476557-5ef8ba42-f36e-4782-be98-2d04b2a447fc.png)
+
+(https://user-images.githubusercontent.com/77942746/148476605-2148befd-6e93-4cde-a363-69ac2561528e.png)
+
+(https://user-images.githubusercontent.com/77942746/148476637-3ab66108-4577-438a-ae78-be2e706a895f.png)
+
```

Comment:
- [ ] These images are appearing on your readme file as raw links.  Let's please kindly correct them, just like you did on line 5.